app/controllers/admin/attachments_controller.rb

```diff
@@ -174,4 +174,12 @@ def handle_duplicate_key_errors_caused_by_double_create_requests(exception)
       raise
     end
   end
+
+  def save_attachment
+    if attachable_is_an_edition?
+      attachment.save && Whitehall.edition_services.draft_updater(attachable).perform!
```

Comment:
Should we just include `PublishesToPublishingApi` on `HtmlAttachment` and then we can just `save`?

I think there's also potentially an issue, albeit a bit of an edge case, with this. If the user saves the attachment on a new draft and then cancels creating the draft it will have been already saved to the publishing API. Whitehall will let users create something else at that base path but publishing API will then conflict.

Reply:
Actually, having discussed this with @davidbasalla offline that issue isn't an issue. The draft edition would have to exist already to get to this point.